elm-app/src/elm/Utils/Html.elm

```diff
@@ -56,3 +60,41 @@ sectionDivider =
 colorToString : Color -> String
 colorToString =
     toString >> toLower
+
+
+formatReceptionDays : Language -> List Day -> String -> String
+formatReceptionDays language days daysDelimiter =
+    if daysDelimiter == "-" then
+        -- Get the first and last day because the "daysDelimiter" tells us that
+        -- the times for all the days are the same.
+        let
+            -- Need to use "case" because List.head returns a Maybe.
+            firstDay =
+                case List.head days of
+                    Just val ->
+                        val
+
+                    Nothing ->
+                        Debug.crash "error: contact reception days are corrupted."
```

Comment:
no `Debug.crash` should be present. If anything, it should be some default value. And you can write it like this:

```elm
firstDay = 
   Maybe.map identifiy (List.head days)
       |> Maybe.withDefault "whatever"
```